pkgs/tools/compression/zstd/default.nix

```diff
@@ -21,8 +21,6 @@ stdenv.mkDerivation rec {
   ];
 
   checkInputs = [ file ];
-  doCheck = false; # fails with "zstd: --list does not support reading from standard input"
```

Comment:
Checks are not enabled by default (unlike python packages), so we need `doCheck = true;` here.
I did that locally and the tests pass, so I think we should enable them.

Reply:
Done!